ipynb/vioscreen_comparisons.ipynb

```diff
@@ -0,0 +1,1009 @@
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "import warnings\n",
+    "warnings.filterwarnings(\"ignore\")\n",
+    "\n",
+    "import numpy as np\n",
+    "import pandas as pd\n",
+    "import skbio\n",
+    "import scipy as sp\n",
+    "import seaborn as sns\n",
+    "from emperor import Emperor, nbinstall\n",
+    "\n",
+    "import matplotlib.pyplot as plt\n",
+    "import matplotlib.gridspec as gridspec\n",
+    "from matplotlib.backends.backend_pdf import PdfPages\n",
+    "% matplotlib inline"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "mapping = pd.read_csv('viscreen_subsample_mapping.txt', sep='\\t') "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "(389, 251)"
+      ]
+     },
+     "execution_count": 3,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "mapping.shape"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "<div>\n",
+       "<style>\n",
+       "    .dataframe thead tr:only-child th {\n",
+       "        text-align: right;\n",
+       "    }\n",
+       "\n",
+       "    .dataframe thead th {\n",
+       "        text-align: left;\n",
+       "    }\n",
+       "\n",
+       "    .dataframe tbody tr th {\n",
+       "        vertical-align: top;\n",
+       "    }\n",
+       "</style>\n",
+       "<table border=\"1\" class=\"dataframe\">\n",
+       "  <thead>\n",
+       "    <tr style=\"text-align: right;\">\n",
+       "      <th></th>\n",
+       "      <th>#SampleID</th>\n",
+       "      <th>types_of_plants</th>\n",
+       "      <th>vioscreen_add_sug</th>\n",
+       "      <th>vioscreen_a_bev</th>\n",
+       "      <th>vioscreen_a_cal</th>\n",
+       "      <th>vioscreen_alcohol_servings</th>\n",
+       "      <th>vioscreen_calcium_avg</th>\n",
+       "      <th>vioscreen_calcium_dose</th>\n",
+       "      <th>vioscreen_calcium_from_dairy_servings</th>\n",
+       "      <th>vioscreen_calcium_servings</th>\n",
+       "      <th>...</th>\n",
+       "      <th>vioscreen_vitd3</th>\n",
+       "      <th>vioscreen_vitd_iu</th>\n",
+       "      <th>vioscreen_vite_iu</th>\n",
+       "      <th>vioscreen_vitk</th>\n",
+       "      <th>vioscreen_water</th>\n",
+       "      <th>vioscreen_wgrain</th>\n",
+       "      <th>vioscreen_xylitol</th>\n",
+       "      <th>vioscreen_zinc</th>\n",
+       "      <th>faiths_pd_1250</th>\n",
+       "      <th>shannon_1250</th>\n",
+       "    </tr>\n",
+       "  </thead>\n",
+       "  <tbody>\n",
+       "    <tr>\n",
+       "      <th>0</th>\n",
+       "      <td>10317.000040</td>\n",
```

Comment:
i'm concerned that these sample IDs are truncated. can this notebook be rerun using data that are not truncated to verify the results dont change?